bump_deb_packaging

```diff
@@ -9,7 +9,8 @@ cd "$PACKAGING_DIR"
 git fetch $PACKAGING_GIT_REMOTE
 git checkout -b deb/release-$PROJECT-$FULLVERSION $PACKAGING_GIT_REMOTE/$REMOTE_BRANCH
 
-scripts/changelog.rb -v ${FULLVERSION/-RC/~rc}-1 -m "$FULLVERSION released" debian/*/*/changelog
+DEBIANIZED_VERSION="$(echo ${FULLVERSION} | sed 's/-RC/~rc/i')"
```

Comment:
Would it be safe to assume we only have lowercase rcs now?

Reply:
you mean, we're not going to do any pre 2.0 RCs anymore? maybe.

Reply:
Exactly. We've never done RCs for GA versions (like a 1.24.2-RC1)